specification/registry/xr.xml

```diff
@@ -10289,10 +10318,10 @@ maintained in the default branch of the Khronos OpenXR GitHub project.
         </require>
     </extension>
 
-    <extension name="XR_OPPO_extension_454" number="454" type="instance" supported="disabled">
+    <extension name="XR_OPPO_controller_interaction" number="454" type="instance" supported="openxr">
         <require>
-            <enum value="1"                                 name="XR_OPPO_extension_454_SPEC_VERSION"/>
-            <enum value="&quot;XR_OPPO_extension_454&quot;" name="XR_OPPO_EXTENSION_454_EXTENSION_NAME"/>
+            <enum value="1"                                 name="XR_OPPO_controller_interaction_SPEC_VERSION"/>
+            <enum value="&quot;XR_OPPO_controller_interaction&quot;" name="XR_OPPO_controller_interaction_EXTENSION_NAME"/>
         </require>
```

Comment:
There's a missing line in here to refer to your interaction profile, see another extension for the format.

Reply:
I added the reference to interaction profile line, following the format of other entries. Let me know if there is anything else I need to update.